Initium-Odp/war/WEB-INF/odppages/viewitemmini.jsp

```diff
@@ -21,6 +21,8 @@
 			<p>${item.itemSlot}</p>
 			<div class='item-popup-stats'>
 				<c:if test="${item.dexpen != null}"><div name='dexterityPenalty' class='item-popup-field' title='This is the percentage that the wearer`s dexterity will be reduced when making dexterity based rolls. Dexterity penalties stack.'>Dexterity penalty: <div class='main-item-subnote'>${item.dexpen}%</div></div></c:if>
+				<c:if test="${item.strmod != null}"><div name='strengthModifier' class='item-popup-field' title='This is the percentage that the wearer`s strength will be modified when making strength based rolls. Strength modifiers stack.'>Strength modifier: <div class='main-item-subnote'>${item.strmod}%</div></div></c:if>
+				<c:if test="${item.intmod != null}"><div name='intelligenceModifier' class='item-popup-field' title='This is the percentage that the wearer`s intelligence will be modified when making intelligence based rolls. Intellgence modifiers stack.'>Intelligence modifier: <div class='main-item-subnote'>${item.intmod}%</div></div></c:if>
 				<c:if test="${item.strReq != null}"><div name='strengthRequirement' class='item-popup-field' title='This is the strength the wearer is required to have in order to equip this item.'>Strength requirement: <div class='main-item-subnote'>${item.strReq}</div></div></c:if>

```

Comment:
Let's put this line first in the list and separate it from the rest with a line break.

Reply:
`<c:if test="${(comparison.strmod != null)||(comparison.intmod != null)}"><br/><br/></c:if>`
This look good?

Reply:
Yep then just put it on top